### ISSUE TITLE
If I follow the default setting and run python main/train.py --cfg configs/cifar10.yaml, I cannot achieve similar results as reported in your paper.At the beginning, the loss of training set decreases, and the accuracy of verification set increases. However, after running for a period of time, the loss of training set will increase, and the accuracy of verification set will decrease, and fluctuate greatly. What's the matter?

### ISSUE THREAD

**Issue by ghost** [Megvii-Nanjing/BBN#19]
If I apply it to my own dataset, the above problems will occur.

**Comment by liangzz1991** [comment]
same question......and find this phenomenon occurs in half of the total epoch.....

**Comment by costantine20** [comment]
do you solve the problem？

**Comment by sunjiaqi401** [comment]
I also have this problem. For a long time, the accuracy of the test set does not rise, and then after such a period of time, it begins to rise again
